src/components/Ayah/index.js

```diff
@@ -37,6 +37,97 @@ export default class Ayah extends Component {
     isSearched: false
   };
 
+  componentDidMount() {
+    
+    function getOffset(elem) {
+      var offsetLeft = 0, offsetTop = 0;
+      do {
+        if ( !isNaN( elem.offsetLeft ) )
+        {
+          offsetLeft += elem.offsetLeft;
+          offsetTop += elem.offsetTop;
+        }
+      } while( elem = elem.offsetParent );
+      return {left: offsetLeft, top: offsetTop};
+    }
+
+    var targets = document.querySelectorAll( '[rel=tooltip]' ),
+    target  = false,
+    tooltip = false,
+    title   = false,
+    tip     = false;
+
+    for(var i = 0; i < targets.length; i++) {
```

Comment:
try mapping over the Array rather than using for loops, its more functional. 
`Array.from(targets).map((item => ...)